cypress/integration/projectSettings/project_settings.ts

```diff
@@ -4,6 +4,8 @@ const getGeneralRoute = (identifier: string) =>
   `${getSettingsRoute(identifier)}/general`;
 const getGithubCommitQueueRoute = (identifier: string) =>
   `${getSettingsRoute(identifier)}/github-commitqueue`;
+const getNotificationsRoute = (identifier: string) =>
```

Comment:
Random comment placement

I get this error:
<img width="595" alt="image" src="https://user-images.githubusercontent.com/13104717/179036442-becd55fd-e58b-4e3a-a4d0-a046599e98e1.png">

when I try to save this: 
<img width="651" alt="image" src="https://user-images.githubusercontent.com/13104717/179036534-d53a5fa4-491c-41c8-b7a2-cd078fd663a2.png">

we should probably just ignore empty ones

Reply:
Great catch! I made the fields required. If they are added.